certbot/crypto_util.py

```diff
@@ -353,3 +354,11 @@ def _notAfterBefore(cert_path, method):
     if six.PY3:
         timestamp_str = timestamp_str.decode('ascii')
     return pyrfc3339.parse(timestamp_str)
+
+
+def sha256sum(filename):
+    """Compute a sha256sum of a file"""
```

Comment:
Since this module is [publicly documented](https://certbot.eff.org/docs/api/crypto_util.html), it'd be nice to document parameters and return values like the rest of the functions in this module.

Our tests and coveralls also think this function doesn't have test coverage because it's only tests are in the Nginx plugin. I think it'd be good to add a simple test or two to `certbot/tests/crypto_util_test.py`.